internal/plugins/ansible/templates/makefile.go

```diff
@@ -0,0 +1,141 @@
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package templates
+
+import (
+	"strings"
+
+	"sigs.k8s.io/kubebuilder/pkg/model/file"
+
+	"github.com/operator-framework/operator-sdk/version"
+)
+
+var _ file.Template = &Makefile{}
+
+// Makefile scaffolds the Makefile
+type Makefile struct {
+	file.TemplateMixin
+
+	// Image is controller manager image name
+	Image string
+
+	// Kustomize version to use in the project
+	KustomizeVersion string
+
+	// AnsibleOperatorVersion is the version of the base image and operator binary used in the project
+	AnsibleOperatorVersion string
+}
+
+// TODO(asmacdo) is this true? if not change everywhere
+// SetTemplateDefaults implements input.Template
+func (f *Makefile) SetTemplateDefaults() error {
+	if f.Path == "" {
+		f.Path = "Makefile"
+	}
+
+	f.TemplateBody = makefileTemplate
+
+	f.IfExistsAction = file.Error
+
+	if f.Image == "" {
+		f.Image = "controller:latest"
+	}
+
+	if f.KustomizeVersion == "" {
+		f.KustomizeVersion = "v3.5.4"
+	}
+
+	if f.AnsibleOperatorVersion == "" {
+		f.AnsibleOperatorVersion = strings.TrimSuffix(version.Version, "+git")
+	}
+
+	return nil
+}
+
+const makefileTemplate = `
+# Image URL to use all building/pushing image targets
+IMG ?= {{ .Image }}
+
+all: docker-build
+
+# Run against the configured Kubernetes cluster in ~/.kube/config
+# TODO(asmacdo)
+# run: ansible-operator
+run: 
+	# TODO(asmacdo) where is ANSIBLE_OPERATOR (HELM_OPERATOR) set?
+	$(ANSIBLE_OPERATOR) run
+
+# Install CRDs into a cluster
+install: kustomize
+	$(KUSTOMIZE) build config/crd | kubectl apply -f -
+
+# Uninstall CRDs from a cluster
+uninstall: kustomize
+	$(KUSTOMIZE) build config/crd | kubectl delete -f -
+
+# Deploy controller in the configured Kubernetes cluster in ~/.kube/config
+deploy: kustomize
+	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
+	$(KUSTOMIZE) build config/default | kubectl apply -f -
+
+# Undeploy controller in the configured Kubernetes cluster in ~/.kube/config
+undeploy: kustomize
+	$(KUSTOMIZE) build config/default | kubectl delete -f -
+
+# Build the docker image
+docker-build:
+	docker build . -t ${IMG}
+
+# Push the docker image
+docker-push:
+	docker push ${IMG}
+
+PATH  := $(PATH):$(PWD)/bin
+SHELL := env PATH=$(PATH) /bin/sh
+OS    = $(shell uname -s | tr '[:upper:]' '[:lower:]')
+ARCH  = $(shell uname -m | sed 's/x86_64/amd64/')
+OSOPER   = $(shell uname -s | tr '[:upper:]' '[:lower:]' | sed 's/darwin/apple-darwin/' | sed 's/linux/linux-gnu/')
+ARCHOPER = $(shell uname -m )
+
+kustomize:
+ifeq (, $(shell which kustomize 2>/dev/null))
+	@{ \
+	set -e ;\
+	mkdir -p bin ;\
+	curl -sSLo - https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/{{ .KustomizeVersion }}/kustomize_{{ .KustomizeVersion }}_$(OS)_$(ARCH).tar.gz | tar xzf - -C bin/ ;\
+	}
+KUSTOMIZE=./bin/kustomize
```

Comment:
From the [Helm plugin](https://github.com/operator-framework/operator-sdk/blob/master/internal/plugins/helm/v1/scaffolds/templates/makefile.go#L118):
```suggestion
KUSTOMIZE=$(realpath ./bin/kustomize)
```